common/include/pcl/common/io.h

```diff
@@ -201,19 +211,25 @@ namespace pcl
   {
     switch (type)
     {
+      case pcl::PCLPointField::BOOL:
+        return 'B';
```

Comment:
```suggestion
        return ('B');
```

Reply:
To keep with others or for some specific reason?

Reply:
Similar to the others.